### PR TITLE
VB interpolated strings topic

### LINKED ISSUE #3547: ### Create VB interpolated strings topic
There should be a VB interpolated strings topic that's comparable to the C# topic.

There is a [preexisting note](https://github.com/dotnet/docs/projects/7#card-2685293) in another project.

### PR DESCRIPTION
**Pull request by rpetrusha**
# VB interpolated strings topic

Fixes #3547 

## Suggested Reviewers

@BillWagner @AnthonyDGreen

### CODE REVIEW

File: docs/visual-basic/language-reference/special-characters/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Special Characters (Visual Basic Reference)"
+ms.date: 10/31/2017
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
```
Comment (BillWagner) [inline docs/visual-basic/language-reference/special-characters/index.md]: This should change to "devlang-visual-basic", right?

File: docs/visual-basic/language-reference/special-characters/interpolated.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "$ (Visual Basic Reference)"
+ms.date: 10/31/2017
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
```
Comment (BillWagner) [inline docs/visual-basic/language-reference/special-characters/interpolated.md]: also devlang-visual-basic

**Review by BillWagner — APPROVED** [review]
This looks great.

I noticed a couple spots in the meta-data where the devlang property mentions C# instead of VB. I'd fix those, and then it's ready to :shipit:

**Comment by rpetrusha** [comment]
Sorry, @BillWagner, I've updated the metadata in this PR, #3546, and not #3549, as I'd commented.

**Pull request merged**